addons/website/static/src/scss/website.scss

```diff
@@ -1378,6 +1378,36 @@ header {
     }
 }
 
+// Step wizard used in /shop, /event, appointment
+.o_wizard {
+    $o_wizard_circle_progress_size: 64px;
+
+    .o_wizard_step-active {
+        @include media-breakpoint-up(md) {
+            color: $body-color;
+        }
+    }
+
+    .o_wizard_circle_progress {
+      width: $o_wizard_circle_progress_size;
+      height: $o_wizard_circle_progress_size;
+      line-height: $o_wizard_circle_progress_size;
+      box-shadow: inset 0px 0px 0px map-get($border-widths, 5) $gray-400;
```

Comment:
`$gray-400` works until the bg is grey 😉
I suggest:

```scss
.o_wizard_circle_progress {
      &, &:before {
          width: $o_wizard_circle_progress_size;
          height: $o_wizard_circle_progress_size;
          line-height: $o_wizard_circle_progress_size;
      }

      &:before {
          @include o-position-absolute(0, 0);
          box-shadow: inset 0 0 0 map-get($border-widths, 5) currentColor;
          opacity: 0.4; // Or "muted" opacity, I let you check
          content: "";
      }
```
Please note `currentColor` and `opacity`... also (minor), you can remove the units from "zero" values ;)